executors/kubeexec/kubeexec.go

```diff
@@ -108,6 +113,12 @@ func NewKubeExecutor(config *KubeConfig) (*KubeExecutor, error) {
 		k.Fstab = config.Fstab
 	}
 
+	if config.MountOpts == "" {
+		k.MountOpts = "rw,inode64,noatime,nouuid"
```

Comment:
We should not have redundant copies of the default string. We should have a single static var for this in the codebase. If its not clear where to put it please ask.

Reply:
@phlogistonjohn Yes please let me know where I need to make this change.

Reply:
@phlogistonjohn I assume I need to add the variable in app.go?

Reply:
Put it in a var in "executors/cmdexec/cmdexec.go" because that's the "parent" of both the kubexec and sshexec implementations.